Week02/lesson04/homework/layoutChallenge/1/styles.css

```diff
@@ -0,0 +1,67 @@
+/*Layout one */
+
+/* Box Model Hack */
+* {
+     -moz-box-sizing: border-box; /* Firexfox */
+     -webkit-box-sizing: border-box; /* Safari/Chrome/iOS/Android */
+     box-sizing: border-box; /* IE */
+}
+
+/* Clear fix hack */
+.clearfix:after {
+     content: ".";
+     display: block;
+     clear: both;
+     visibility: hidden;
+     line-height: 0;
+     height: 0;
+}
+
+.clear {
+  clear: both;
+}
+
+/*layout */
+
+body {
+  width: 100%;
+}
+
+#container{
+  width: 1000px;
```

Comment:
It's totally fine that you went with a fixed-pixel approach it that helps you understand CSS layouts better, but you may want to consider a percentage-based with the `container` being `width: 100%;` and the 
`article` and `aside` being around 60% and 40%